block/src/Pos/DB/Block.hs

```diff
@@ -107,25 +105,26 @@ putSerializedBlunds
     :: (HasConfiguration, MonadRealDB ctx m, MonadDB m)
     => NonEmpty SerializedBlund -> m ()
 putSerializedBlunds (toList -> bs) = do
-    bdd <- view blockDataDir <$> getNodeDBs
-    let allData = map (\(b,u) -> let (dP, bP, uP) = getAllPaths bdd (headerHash b)
-                                 in (dP,(b,u,bP,uP))
+    let allData = map (\(b,u) -> let hh = formatToString hashHexF (headerHash b)
```

Comment:
Why `formatToString` and not `format`?
Also I would move `encodeUtf8` to `blockKeyMod` and `undoKeyMod`, because they are always used in composition.

Reply:
Fair enough, I'll do that.